docs/reference-guides/data/data-core.md

```diff
@@ -329,6 +329,39 @@ _Returns_
 
 -   `number | null`: number | null.
 
+### getEntityRevision
```

Comment:
This name is not ideal. In the sense that we're not retrieving the revision of an entity, we're retrieving a revision of an entity record. So either it should be something like `getEntityRecordRevision` or maybe just `getRevision` (same for the other selector